content/md/first-steps/manage-data-quality.md

```diff
@@ -0,0 +1,30 @@
+---
+id: manage-data-quality
+themes: catalog-settings
+title: Manage your **data quality**
+popular: false
+ee-only: true
+related: understand-data-quality, improve-data-quality
+---
+
+# Overview
+
+In order to check, score and get recommendations on the product data that truly matters to craft a great product experience,  you can disable Data Quality Insights calculation for non relevant attribute groups (ie. ERP, internal…)
+By doing so, you will be able to highlight the quality of the attributes that are the most valuable for your customers.
+
+# Data quality insights activation and deactivation
+
+:::info
+You must be granted a specific permission to be able to edit attribute groups.
```

Comment:
What do they need to do? ask admin? or set it themselves? might be worth mentioning!

Reply:
changes made